core/citrus-base/src/main/java/org/citrusframework/actions/ReceiveMessageAction.java

```diff
@@ -336,11 +366,14 @@ private void assumeMessageType(Message message) {
 
                 setMessageType(MessageType.YAML);
             }
+        } else {
+            setMessageType(MessageType.PLAINTEXT);
         }
     }
 
     /**
      * Create control message that is expected. Apply global and local message processors and data dictionaries.
+     *
```

Comment:
All this javadoc is useless. We used to remove such zero information doc in the past. You may want to stick with that.

Reply:
done. class cleaned up.